stable/rabbitmq-ha/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,69 @@
+{{- if .Values.networkPolicy.enable }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "rabbitmq-ha.fullname" . }}
+  labels:
+    app: {{ include "rabbitmq-ha.fullname" . }}
+    name: {{ include "rabbitmq-ha.name" . }}
+    fullname: {{ include "rabbitmq-ha.fullname" . }}
+    chart: {{ include "rabbitmq-ha.chart" . }}
+    app.kubernetes.io/name: {{ include "rabbitmq-ha.fullname" . }}
```

Comment:
Don't mix and match old labels and new. We should match what the rest of the chart uses:
```yaml
  labels:
    app: {{ template "rabbitmq-ha.name" . }}
    chart: {{ template "rabbitmq-ha.chart" . }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
```